lib/lotus/generators/generatable.rb

```diff
@@ -46,6 +46,11 @@ def template_source_path
       def target_path
         raise NotImplementedError, "must implement the target_path method"
       end
+
+      def argument_blank?(value)
+        value.nil? || value.strip.empty?
```

Comment:
@accessd Can you please replace it with `Lotus::Utils::String.new(value).blank?`? This method has a better support for UTF-8 chars. Thank you :+1: